.github/workflows/publish-autoinstrumentation-java.yaml

```
@@ -37,6 +37,7 @@ jobs:
             ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-java
           tags: |
             type=match,pattern=v(.*),group=1,value=v${{ env.VERSION }}
+            type=semver,pattern={{major}},value=v${{ env.VERSION }}
```

Shouldn't this be
```suggestion
            type=semver,pattern=v{{major}},value=v${{ env.VERSION }}
```
? Otherwise we'd get just `1`, and I think we want `v1`.

---

Or maybe it should be `1.x`?

---

Ya it can be whatever we want

---

since `v1` is defined in the readme's example I'd choose that over `1.x`.

---

https://github.com/open-telemetry/opentelemetry-operator/pkgs/container/opentelemetry-operator%2Fautoinstrumentation-java

---

at the moment we don't prefix with `v`